### PR TITLE
remove racism

### PR DESCRIPTION
**Pull request by LankLTE**
## About the PR

Technically speceism or whatever but I wanted the funny title. Removes the AI storm rules that say "Only [species] is part of the crew". 

This can no longer happen: 
image

## Why / Balance

Discrimination based on species is explicitly against the server rules, so telling a borg that they have to genocide all moths or something is bound to cause issues. I'm sure it could be funny, but I don't imagine admins want to deal with an ahelp like "why are you screaming kill all lizards" "my laws told me to" or something.
The comment also *explicitly says* it's for species genocide, which makes me think it was just ported from 13 and doesn't really have a place here. 

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by mirrorcult** [comment]
we did it guys

**Comment by IProduceWidgets** [comment]
> entities with HumanoidAppearance do "osay $ID Emote claps"

File: Content.Server/StationEvents/Events/IonStormRule.cs [hunk 1]
```diff
@@ -206,21 +203,20 @@ private string GenerateLaw()
...
+            4 => crew1,
             _ => Loc.GetString("ion-storm-x-and-y", ("x", crew1), ("y", crew2))
         };
 
         if (plural) feeling = feelingPlural;
 
         // message logic!!!
-        return RobustRandom.Next(0, 37) switch
+        return RobustRandom.Next(0, 36) switch
         {
```
Comment (metalgearsloth) [inline Content.Server/StationEvents/Events/IonStormRule.cs:220]: Which psycho coded this to not just use a string array and length?

Comment (LankLTE) [reply Content.Server/StationEvents/Events/IonStormRule.cs:220]: delta

Comment (deltanedas) [reply Content.Server/StationEvents/Events/IonStormRule.cs:220]: me

**Comment by JerryImMouse** [comment]
this racism is based on IC reason, i dont think it should be removed.

**Comment by metalgearsloth** [comment]
> this racism is based on IC reason, i dont think it should be removed.

IC racism is not allowed.

File: Content.Server/StationEvents/Events/IonStormRule.cs (hunk 1 above)
Comment (metalgearsloth) [reply Content.Server/StationEvents/Events/IonStormRule.cs:220]: it's dm so I'd believe it.

**Comment by lzk228** [comment]
bruh

**Comment by Hmeister-fake** [comment]
This law is very funny. God rest o7

**Comment by metalgearsloth** [comment]
> bruh

This has been a rule on servers for years.

**Comment by JerryImMouse** [comment]
> IC racism is not allowed.

Maybe just make it toggleable to allow downstreams turn it on if they wish?

**Comment by metalgearsloth** [comment]
> > IC racism is not allowed.
> 
> Maybe just make it toggleable to allow downstreams turn it on if they wish?

Downstreams can choose what to merge.

**Comment by deltanedas** [comment]
play on tg for racism :trollface:

**Pull request merged**